### DISCUSSION TITLE
voiceserver php8.x upgrade please for ilch 2.1.51

### DISCUSSION THREAD

**Discussion by BaehrlyBahr** [community/community#61396] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

Hallo can you program voiceserver php8.x upgrade please for ilch 2.1.51 please need some updated version for the new php version.
thx for your great work on voiceserver module.
mfg baehrlybahr

**Comment by dannz510** [comment]
You may be able to find resources or developers who can help you upgrade the voiceserver module for ilch 2.1.51 to be compatible with PHP 8.x. You can try reaching out to the ilch community or searching for developers with experience in PHP and ilch module development.
